pkg/gcv/configs/config.go

```diff
@@ -103,7 +104,7 @@ func ListRegoFiles(dir string) ([]string, error) {
 	return arrayFilterSuffix(files, ".rego"), nil
 }
 
-func loadUnstructured(dirs []string) ([]*unstructured.Unstructured, error) {
+func LoadUnstructured(dirs []string) ([]*unstructured.Unstructured, error) {
```

Comment:
can we add go doc comments for these exported funcs?

Reply:
done